cluster-operator/src/test/java/io/strimzi/operator/cluster/model/ModelUtilsTest.java

```diff
@@ -302,61 +312,166 @@ public void testValidHeapPercentage() {
     }
     
     @ParallelTest
-    public void testJvmMemoryOptionsExplicit() {
+    void testJvmMemoryOptionsExplicit() {
         Map<String, String> env = heapOptions(jvmOptions("4", "4"), 50, 4_000_000_000L, null);
         assertThat(env.get(AbstractModel.ENV_VAR_KAFKA_HEAP_OPTS), is("-Xms4 -Xmx4"));
         assertThat(env.get(AbstractModel.ENV_VAR_DYNAMIC_HEAP_PERCENTAGE), is(nullValue()));
         assertThat(env.get(AbstractModel.ENV_VAR_DYNAMIC_HEAP_MAX), is(nullValue()));
     }
 
     @ParallelTest
-    public void testJvmMemoryOptionsXmsOnly() {
+    void testJvmMemoryOptionsXmsOnly() {
         Map<String, String> env = heapOptions(jvmOptions(null, "4"), 50, 5_000_000_000L, null);
         assertThat(env.get(AbstractModel.ENV_VAR_KAFKA_HEAP_OPTS), is("-Xms4"));
         assertThat(env.get(AbstractModel.ENV_VAR_DYNAMIC_HEAP_PERCENTAGE), is(nullValue()));
         assertThat(env.get(AbstractModel.ENV_VAR_DYNAMIC_HEAP_MAX), is(nullValue()));
     }
 
     @ParallelTest
-    public void testJvmMemoryOptionsXmxOnly() {
+    void testJvmMemoryOptionsXmxOnly() {
         Map<String, String> env = heapOptions(jvmOptions("4", null), 50, 5_000_000_000L, null);
         assertThat(env.get(AbstractModel.ENV_VAR_KAFKA_HEAP_OPTS), is("-Xmx4"));
         assertThat(env.get(AbstractModel.ENV_VAR_DYNAMIC_HEAP_PERCENTAGE), is(nullValue()));
         assertThat(env.get(AbstractModel.ENV_VAR_DYNAMIC_HEAP_MAX), is(nullValue()));
     }
 
     @ParallelTest
-    public void testJvmMemoryOptionsDefaultWithNoMemoryLimitOrJvmOptions() {
+    void testJvmMemoryOptionsDefaultWithNoMemoryLimitOrJvmOptions() {
         Map<String, String> env = heapOptions(jvmOptions(null, null), 50, 5_000_000_000L, null);
         assertThat(env.get(AbstractModel.ENV_VAR_KAFKA_HEAP_OPTS), is("-Xms" + ModelUtils.DEFAULT_JVM_XMS));
         assertThat(env.get(AbstractModel.ENV_VAR_DYNAMIC_HEAP_PERCENTAGE), is(nullValue()));
         assertThat(env.get(AbstractModel.ENV_VAR_DYNAMIC_HEAP_MAX), is(nullValue()));
     }
 
     @ParallelTest
-    public void testJvmMemoryOptionsDefaultWithMemoryLimit() {
+    void testJvmMemoryOptionsDefaultWithMemoryLimit() {
         Map<String, String> env = heapOptions(jvmOptions(null, "4"), 50, 5_000_000_000L, new ResourceRequirementsBuilder().withLimits(Map.of("memory", new Quantity("1Gi"))).build());
         assertThat(env.get(AbstractModel.ENV_VAR_KAFKA_HEAP_OPTS), is("-Xms4"));
         assertThat(env.get(AbstractModel.ENV_VAR_DYNAMIC_HEAP_PERCENTAGE), is("50"));
         assertThat(env.get(AbstractModel.ENV_VAR_DYNAMIC_HEAP_MAX), is("5000000000"));
     }
 
     @ParallelTest
-    public void testJvmMemoryOptionsMemoryRequest() {
+    void testJvmMemoryOptionsMemoryRequest() {
```

Comment:
Maybe all these changes can be removed from the PR to minimize unnecessary changes and focus on a single task in it?

Reply:
I'll extract this to proposed J`vmOptionsUtils` class and corresponding `JvmOptionsUtilsTest` and revert changes in here.

Reply:
I've moved all `JvmOptions` related tests to `JvmOptionsUtilsTest` and restored other changes back to original.